### PR TITLE
Main + Team pages

### LINKED ISSUE #40: [App] Main page + About page
- [ ] Main page
- [ ] About page

image

image

image

### PR DESCRIPTION
**Pull request by superpuper777**

### CODE REVIEW

File: rslang/src/app/navigation/navigation.module.ts [hunk 1]
```diff
@@ -1,15 +1,41 @@
...
-  exports: [NavMenu, Settings, DotMenu, SettingsDialog],
+  imports: [
+    CommonModule,
+    RouterModule,
+    MatIconModule,
+    MatMenuModule,
+    MatToolbarModule,
+    MatDialogModule,
+    MatCheckboxModule,
+    MatButtonModule,
```
Comment (Nikitzu) [inline rslang/src/app/navigation/navigation.module.ts]: not here

**Review by Nikitzu — CHANGES_REQUESTED** [review]

File: rslang/src/app/components/main/main.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    </iframe>
+  </div>
+</section>
+<section class="main__team">
+  <mat-accordion>
+    <mat-expansion-panel (opened)="panelOpenState = true" (closed)="panelOpenState = false">
+      <mat-expansion-panel-header>
+        <!-- <mat-panel-title> OUR TEAM </mat-panel-title> -->
+        <mat-panel-description>
+          <h3 class="main__our__team__title">Click to show our team</h3>
```
Comment (Nikitzu) [inline rslang/src/app/components/main/main.html]: h3 with text click? And where is the click itself?

**Review by Nikitzu — APPROVED** [review]

**Review by Nikitzu — CHANGES_REQUESTED** [review]
One comment more

File: rslang/src/app/components/main/main.html (hunk 2 above)
Comment (superpuper777) [reply rslang/src/app/components/main/main.html]: You need to click on the expansion panel to open it.

**Review by Nikitzu — APPROVED** [review]

**Review by Anntol — APPROVED** [review]

**Pull request merged**